ScrapPackedExplorerCli/ScrapPackedExplorerCli.csproj

```diff
@@ -47,4 +47,6 @@
     </Page>
   </ItemGroup>
 
+  <ProjectExtensions><VisualStudio><UserProperties properties_4launchsettings_1json__JsonSchema="" /></VisualStudio></ProjectExtensions>
+
```

Comment:
no need to answer …
no need to remove …
… just wondering if you know what this is :D

Reply:
Wait, isn't it yours? I got it after fetching your repo